src/shared/responseHelper.js

```diff
@@ -0,0 +1,69 @@
+class ResponseHelper{
+    /**
+     * @function
+     * @description A simple API response helper
+     *
+     * @param {Object} res Express response object
+     * @param {Number} status HTTP status code
+     * @param {Boolean} error Flag for error
+     * @param {Object} data Response data object
+     * @param {String} message Response message to client
+     *
+     * @returns {Object} Response from the server
+     */
+
+    static sendResponse(
+        res,
+        status,
+        error = false,
```

Comment:
I think this is not needed, it is kind of redundant, instead what would be ok is having an error object